apps/web/pages/apps/onboarding/[[...step]].tsx

```diff
@@ -512,15 +512,20 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
         if (!hasEventTypes) {
           throw new Error(ERROR_MESSAGES.appNotExtendsEventType);
         }
-        eventTypes = await getEventTypes(user.id, parsedTeamIdParam);
+        const _eventTypes = await getEventTypes(user.id, parsedTeamIdParam);
+
+        if (!_eventTypes) {
+          return { redirect: { permanent: false, destination: `/apps/installed/${appMetadata.category}` } };
```

Comment:
if user/team does not have event-type then skip configure step and goto apps/installed/[category]